test/common/run-tests

```diff
@@ -65,14 +65,25 @@ def print_test(test, print_tap=True):
         print("not ok " + test_name(test))
     sys.stdout.flush()
 
-def finish_test(opts, test):
+def finish_test(opts, test, affected_tests):
     """Returns if a test should retry or not
 
     Call test-policy on the test's output, print if needed.
 
     Return (retry_reason, exit_code). retry_reason can be None or a string.
     """
+
+    affected = len(affected_tests) < 3 and test.command[0].encode("utf-8") in affected_tests
+
     if test.process.returncode in [0, 77]:
+        # Try affected tests 3 times
+        if affected and test.retries < 2:
+            retry_reason = b"test affected tests 3 times"
+            test.retries += 1
+            test.output += b" # RETRY %i (%s)\n" % (test.retries, retry_reason)
```

Comment:
Not super sure if we want it as "RETRY" as it probably will be shown on the top of the html test output, which may seem like failed tests. (Or maybe we parse not based on RETRY, but on "not ok"?, lets see how the output looks like).

Reply:
tests-policy, store-failures, and log.html should all first look at `not ok`. I. e. they have code to do so, it *may* be buggy of course :) But should be alright, let's try.

Reply:
Indeed, it works fine.